docker-app/qfieldcloud/core/serializers.py

```diff
@@ -277,9 +283,7 @@ def to_internal_value(self, data):
             if self._choices[i] == data:
                 return i
         raise serializers.ValidationError(
-            "Invalid status. Acceptable values are {}.".format(
-                list(self._choices.values())
-            )
+            "Invalid status. Acceptable values are {}.".format(self._choices.values())
```

Comment:
Why is this change here? Please revert to the original state.

I am afraid I mention this problem here for the third time...

Reply:
Solved (for real)